src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -113,11 +113,15 @@
             new String[] {String.valueOf(1)}, null, getClass(), null);
 
     private static String getPath(String filename) {
-        return "src/test/resources/com/puppycrawl/tools/checkstyle/" + filename;
+        return "src/test/resources/com/puppycrawl/tools/checkstyle/main/" + filename;
+    }
+
+    private static String getNonPackagePath(String filePath) {
+        return "src/test/resources/com/puppycrawl/tools/checkstyle/" + filePath;
```

Comment:
@Kietzmann Every comment I made below discusses that. These files shouldn't be in outer scope but be in MainTest's input folder.